src/pages/EntityDetails/Model/Model.test.tsx

```diff
@@ -0,0 +1,521 @@
+import configureStore from "redux-mock-store";
+import { Provider } from "react-redux";
+import { QueryParamProvider } from "use-query-params";
+import { ReactRouter6Adapter } from "use-query-params/adapters/react-router-6";
+import { MemoryRouter, Route, Routes } from "react-router-dom";
+import { render, screen, within } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import dataDump from "testing/complete-redux-store-dump";
+
+import { reduxStateFactory } from "testing/redux-factory";
+import { ReduxState } from "types";
+import { ModelData } from "juju/types";
+
+import Model, { Label } from "./Model";
+import { TestId } from "../../../components/InfoPanel/InfoPanel";
+
+jest.mock("components/Topology/Topology", () => {
+  const Topology = () => <div className="topology"></div>;
+  return Topology;
+});
+
+jest.mock("components/WebCLI/WebCLI", () => {
+  const WebCLI = () => <div className="webcli" data-testid="webcli"></div>;
+  return WebCLI;
+});
+
+jest.mock("juju", () => {
+  return {
+    queryOperationsList: () => {
+      return new Promise((resolve) => {
+        const apiData = require("testing/list-operations-api-response.json");
+        resolve(apiData.response);
+      });
+    },
+    queryActionsList: () => {
+      return new Promise((resolve) => {
+        const apiData = require("testing/list-actions-api-response.json");
+        resolve(apiData.response);
+      });
+    },
+  };
+});
+
+const mockStore = configureStore([]);
+
+describe("Model", () => {
```

Comment:
These tests were skipped for some reason. I've done a rough update to RTL but this really needs the factories before it can be cleaned up further.